test/vmtests/vmtests/imagecreator/test_imagecreator.py

```diff
@@ -185,36 +190,113 @@ def run_image_creator_test(
     logging.info(f"Attempting to connect to VM via SSH")
     logging.info(f"Username: {username}")
 
-    with vm.create_ssh_client(ssh_private_key_path, test_temp_dir, username) as ssh_client:
-        logging.info(f"SSH connection established successfully!")
-        # Run the test
-        logging.info(f"Running basic checks on the VM")
-        run_basic_checks(ssh_client, test_temp_dir)
-        logging.info(f"Basic checks completed successfully!")
+    try:
+        with vm.create_ssh_client(ssh_private_key_path, test_temp_dir, username) as ssh_client:
+            logging.info("SSH connection established successfully!")
+            # Test SSH connectivity and shell access
+            result = ssh_client.run("whoami")
+            if result.exit_code != 0:
+                logging.error("Failed to execute basic SSH command")
+                logging.error(f"Command output: {result.stdout}")
```

Comment:
The `ssh_client.run` already logs everything. You don't need to log it again. If you want to log at a higher level, then use the `stdout_log_level` and `stderr_log_level` parameters.

Same comment elsewhere.